src/mocks/handlers.ts

```diff
@@ -1,33 +1,66 @@
 import { rest } from "msw";
 
+export const resourceToMockMap = {
```

Comment:
Used in `cy.mock` so that we can mock the same request both on the client and server